packages/server/lib/db/db.config.ts

```diff
@@ -14,6 +14,10 @@ let dbConfig: { development: Knex.Config<any>; production: Knex.Config<any> } =
         migrations: {
             directory: './migrations',
             extension: 'ts'
+        },
+        pool: {
+            min: process.env['NANGO_DB_POOL_MIN'] || 2,
+            max: process.env['NANGO_DB_POOL_MAX'] || 7
```

Comment:
this code seemed trivial, so I didn't actually test it. Let me know if you want me to test it or there are some automated checks already.

Reply:
Don't think there are any actual tests. But, it builds.